azure-pipelines.yml

```diff
@@ -0,0 +1,25 @@
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/resources?view=azure-devops&tabs=schema#define-a-repositories-resource
+resources:
+  repositories:
+    - repository: IOMDevopsRepo
+      type: git
+      name: order-iom-partner-devops
+      ref: task/76533-ci-pipeline
+
+trigger:
+  branches:
+    include:
+      - '*'
+
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#extend-from-a-template
+# see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops#use-other-repositories
+extends:
+  template: template.yml@IOMDevopsRepo
```

Comment:
Good idea to extend from a compony owned template.
These parameters are defined by us and are not reserved words, right ?

Reply:
Right